packages/paste-core/components/spinner/src/constants.ts

```diff
@@ -0,0 +1,11 @@
+type CircleGeometryProps = Pick<React.SVGProps<SVGCircleElement>, 'cx' | 'cy' | 'r'>;
+
+const circleRadius = 45;
+export const circleGeometry: CircleGeometryProps = {
+  cx: 50,
+  cy: 50,
+  r: circleRadius,
+};
+export const circleCircumference = Math.floor(Math.PI * 2 * circleRadius);
```

Comment:
Can we add some comments here so that future maintainers know how to reason through this code please?